### PR TITLE
The server code is meant to be only for testing/development

### PR DESCRIPTION
**Pull request by itsthatguy**
@rylnd Does this address your request in #90?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,9 +8,9 @@
   "scripts": {
     "test": "mocha --exit",
     "build": "babel src/ --out-dir=dist/ --copy-files",
-    "dev": "concurrently --kill-others --prefix=name --names=server,eslint --prefix-colors=green,magenta \"babel-watch --watch src src/server.js\" \"esw src/ --changed --watch\"",
-    "start": "babel-node src/server.js",
-    "start:prod": "node dist/server.js",
+    "dev": "concurrently --kill-others --prefix=name --names=server,eslint --prefix-colors=green,magenta \"npm run start:server\" \"npm run start:lint\"",
```
Comment (rylnd) [inline package.json]: Should these be `dev:server` and `dev:lint`?

File: README.md [hunk 2]
```diff
@@ -71,17 +71,17 @@ Then, there are several npm scripts that will be useful:
...
-npm start
-
 # run both a dev server and eslint
 npm run dev
 
+# run a dev server
+npm run dev:server
+
+# run eslint
+npm run dev:server
```
Comment (rylnd) [inline README.md]: `dev:lint`

**Comment by rylnd** [comment]
@itsthatguy it's a good start. I assume that you moved the server file out of `src/` so that it wouldn't be transpiled to `dist/`, but it's a bit weird for the target of `dev:server` to live in the `test/` folder. I think that moving server.js to the root would be a bit less surprising and would accomplish the same goals, but that's a guess that needs verification.

**Comment by itsthatguy** [comment]
@rylnd I moved it to `test/` to very clearly communicate that it only exists for test purposes.

**Comment by rylnd** [comment]
Fair enough! I'm down with that.

The only outstanding thing I see, then, are references to our old heroku deployment stuff:

* `heroku-postbuild` script
* Procfile
* .buildpacks

I'm happy to remove those quick, or you can; lemme know.

**Comment by itsthatguy** [comment]
@rylnd DONE!

**Pull request merged**